umbraco/Umbraco-CMS
# New backoffice: Change management API base path

Changes the new backoffice management api base path from: `/umbraco/api/` to `/umbraco/management/api/`

I've also ensured that all the controller bases now uses the `VersionedApiBackOfficeRoute` routing attribute and that two words are separated by `-` instead of camel case (when it is not a param name).


## Testing
1. Run the site
2. Ensure that all management API paths start with `/umbraco/management/api/`

The easiest way to do this is to visit the `/umbraco/swagger` endpoint
